src/AptabasePersistentClient.cs

```diff
@@ -19,6 +18,8 @@ public class AptabasePersistentClient : IAptabaseClient
     private readonly ILogger<AptabasePersistentClient>? _logger;
     private readonly CancellationTokenSource _cts;
 
+    private bool _pauseProcessing;
```

Comment:
Can we use the CancellationTokenSource to stop sending events? Maybe add a method to the IAptabaseClient interface Stop, which calls `_cts.Stop()`?

Reply:
No, because I don't think we want to actually stop the ProcessEvents task in this case. If for some reason the app doesn't actually crash, then that will stop sending events completely. For example, in Android another UncaughtException listener can set ue.Handled, and the app will keep going. Likewise if an "ApplicationCrash" event is tracked for whatever reason. The 30-second pause seems (to me) like a good option - it's certainly simple and predictable, and keeps the library out of the way.

Reply:
I understand, the thing that bothers me about this approach, is that we're leaking crash specific details into the persistent client implementation. I'm wondering if we can find a way to pause/start the client, maintaining the separation